app/models/katello/sync_plan.rb

```diff
@@ -34,7 +33,20 @@ class SyncPlan < Katello::Model
     scoped_search :on => :name, :complete_value => true
     scoped_search :on => :organization_id, :complete_value => true, :only_explicit => true, :validator => ScopedSearch::Validators::INTEGER
     scoped_search :on => :interval, :complete_value => true
-    scoped_search :on => :enabled, :complete_value => true
+    scoped_search :on => :enabled, :complete_value => { :true => true, :false => false }, :ext_method => :search_enabled
+
+    def self.search_enabled(_key, operator, value)
+      value = ::Foreman::Cast.to_bool(value)
+      value = !value if operator == '<>'
+      active_logics = ForemanTasks::RecurringLogic.where(:state => "active")
+      active_logic_ids = active_logics.pluck(:id)
+      if active_logic_ids.empty?
+        {:conditions => "1=0"}
+      else
+        operator = value ? 'IN' : 'NOT IN'
+        {:conditions => "#{Katello::SyncPlan.table_name}.foreman_tasks_recurring_logic_id #{operator} (#{active_logic_ids.join(',')})"}
+      end
+    end
```

Comment:
mind adding a test for this search?

Reply:
Added [here](https://github.com/Katello/katello/pull/8008/files#diff-4eab861b3ddf949cb85847481a8eb18aR362)..